pkg/network/telemetry/ebpf_telemetry.go

```diff
@@ -38,17 +39,21 @@ var helperNames = map[int]string{readIndx: "bpf_probe_read", readUserIndx: "bpf_
 // EBPFTelemetry struct contains all the maps that
 // are registered to have their telemetry collected.
 type EBPFTelemetry struct {
-	MapErrMap    *ebpf.Map
-	HelperErrMap *ebpf.Map
-	mapKeys      map[string]uint64
-	probeKeys    map[string]uint64
+	MapErrMap             *ebpf.Map
+	HelperErrMap          *ebpf.Map
+	mapKeys               map[string]uint64
+	probeKeys             map[string]uint64
+	ebpfMapOpsErrorsGauge telemetry.Gauge
+	ebpfHelperErrorsGauge telemetry.Gauge
 }
 
 // NewEBPFTelemetry initializes a new EBPFTelemetry object
 func NewEBPFTelemetry() *EBPFTelemetry {
 	return &EBPFTelemetry{
-		mapKeys:   make(map[string]uint64),
-		probeKeys: make(map[string]uint64),
+		mapKeys:               make(map[string]uint64),
+		probeKeys:             make(map[string]uint64),
+		ebpfMapOpsErrorsGauge: telemetry.NewGauge("ebpf_map_ops", "errors", []string{"map_name", "error"}, "Failures of map operations for a specific ebpf map reported per error."),
```

Comment:
Based on prometheus naming guidelines, I think this should be `errors_total`. https://prometheus.io/docs/practices/naming/